src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -466,6 +466,12 @@ public void BeginBuild(BuildParameters parameters)
 
             _previousLowPriority = parameters.LowPriority;
 
+            if (Traits.Instance.DebugEngine)
+            {
+                parameters.DetailedSummary = true;
+                parameters.LogTaskInputs = true;
```

Comment:
What other implications does setting summary here have? Does it affect attached loggers?